internal/certdb/certdb.go

```diff
@@ -0,0 +1,32 @@
+package certdb
+
+import (
+	"fmt"
+
+	"github.com/test-network-function/cnf-certification-test/internal/certdb/offlinecheck"
+	"github.com/test-network-function/cnf-certification-test/internal/certdb/onlinecheck"
+	"helm.sh/helm/v3/pkg/release"
+)
+
+//go:generate moq -out api_moq.go . CertAPIClientFuncs
+type ResourceCertificator interface {
```

Comment:
This is a personal opinion, but "Certificator" suffix seems a bit misleading to me, as we're not certifiying any resources, just getting their certification status. I would stick to "StatusChecker" or "CertificationStatusValidator". I see you also changed some function receiver names from "checker" to "validator", so maybe the second option is better for consistency.